### PR TITLE
Added scenario 12 with land-use requirements

### LINKED ISSUE #147: Add new scenario w/ land-use requirements
This issue can be closed when a new scenario:
* removes the ``` CostVariable ``` table
* removes the ``` CostFixed ``` table
* replaces the entries of ``` CostInvest ``` with the specific land-use requirements (land-use/capacity)

The table ``` CostInvest ``` should have entries for at least the following technologies:
- [x] ``` IMPSOL ```: solar power
- [x] ``` IMPWIND ```: wind power
- [x] ``` NUCLEAR ```
- [x] ``` GEOT ```: geothermal heat pump

If time allows, the table should also have entries for the following technologies:
- [x] ``` ABBOTT ```: Abbott power plant
- [x] ``` CHWS ```: Chilled water storage
- [ ] ``` ELECTROL ```: Electrolysis plant

### PR DESCRIPTION
**Pull request by robfairh**
Fixes #147

### CODE REVIEW

File: data_processing/landuse.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Land Use Requirements
```
Comment (yardasol) [inline data_processing/landuse.md:1]: Very nice!!

File: temoa-uiuc/data_files/12_uiuc.sql [hunk 2]
```diff
@@ -0,0 +1,1436 @@
...
+	"tech"	text,
+	"vintage"	integer,
+	"cost_invest"	real,
+	"cost_invest_units"	text,
+	"cost_invest_notes"	text,
+	FOREIGN KEY("tech") REFERENCES "technologies"("tech"),
+	FOREIGN KEY("vintage") REFERENCES "time_periods"("t_periods"),
+	PRIMARY KEY("regions","tech","vintage")
+);
+-- UIUC Data
```
Comment (yardasol) [inline temoa-uiuc/data_files/12_uiuc.sql:1043]: this is where you added the land use data, correct?

**Review by yardasol — APPROVED** [review]

File: temoa-uiuc/data_files/12_uiuc.sql (hunk 2 above)
Comment (robfairh) [reply temoa-uiuc/data_files/12_uiuc.sql:1043]: yes

**Review by samgdotson — APPROVED** [review]
Results are consistent with my expectations w.r.t power density. 

Merging

File: data_processing/landuse.md [hunk 3]
```diff
@@ -14,11 +20,26 @@
 
 * CHWS: 15 Ton/m^2 if size > 20,000 Ton [4] ~ 0.066 m^2/Ton
 
+* Nuclear(SMRs) [6]:
```
Comment (yardasol) [inline data_processing/landuse.md:16]: Nice data!

**Pull request merged**